modules/apps/portal-vulcan/portal-vulcan-api/src/test/java/com/liferay/portal/vulcan/util/LocalizedMapUtilTest.java

```diff
@@ -36,6 +36,55 @@ public class LocalizedMapUtilTest {
 	public static final LiferayUnitTestRule liferayUnitTestRule =
 		LiferayUnitTestRule.INSTANCE;
 
+	@Test
+	public void testGetI18nMapWithAcceptAllLanguagesAndAvailableLocalesAndLocalizedMap() {
```

Comment:
can we do just a single test `testGetI18nMap`? For example the `testMergeI18nMap` seems to test all the "possible parameter combination" in a single one. 

Also they seems to do the same thing: I mean... if in all the cases we have the same locales how can we check the functionality? I mean... for the `testGetI18nMapWithAcceptAllLanguagesAndAvailableLocalesAndLocalizedMap` I think we should have different availableLocales compared with the map we are passing, otherwise how we can know that it is filtering them? I don't know for the other what should be the difference between allLanguages true or false

Reply:
@smottal They are not the same. 1-2 param method does basically the same, so that is okay, but there is no filtering there. 3 param is the only one that filters, so at least 2 && 3 param or 1 && 3 param needs to be kept. As for 1 and 2 param, in that sense you are right that it could be enough to just test 1 param since that calls 2 param method with default param acceptAllLang=true. So, I still suggest to keep 1 and 3 param test methods. Testing 2 param with acceptAllLanguages=false does not make any sense for me because it will return immediately with null, I think it is not necessary to test bc it is too simple.

Reply:
@smottal They could all be merged to 1 test, but I dont think that's good practice to test different methods with 1 test. I think this way it is clear at first look which one I'm testing.

Reply:
@smottal Brian also asked "Please add some basic tests for all the getI18nMap methods in LocalizedMapUtil". I hope I managed to convince you. :)